ArduCopter/GCS_Mavlink.cpp

```diff
@@ -363,8 +363,7 @@ bool GCS_MAVLINK_Copter::try_send_message(enum ap_message id)
 #if RANGEFINDER_ENABLED == ENABLED
         CHECK_PAYLOAD_SIZE(RANGEFINDER);
         send_rangefinder_downward(copter.rangefinder);
-        CHECK_PAYLOAD_SIZE(DISTANCE_SENSOR);
-        send_distance_sensor_downward(copter.rangefinder);
+        send_distance_sensor(copter.rangefinder);
```

Comment:
We can't do this without more changes. The reason we kept just sending the downward sensor here is that the proximity will send all the other directions if using rangefinder so it would have a duplication of messages. I know there is a (logging) use case for having rangefinders in other directions and not being used by proximity, but it's certainly an unusual one.

Reply:
That's only true if proximity is configured as `Proximity_Type_RangeFinder` otherwise they are different values. I could special case that, but it starts to feel problematic, I admit.

Reply:
Yes, but if you don't have Proximity_Type_RangeFinder set then all the other directions are only doing logging, which is the unusual case I wrote above.

Reply:
Right, I guess I'm arguing that it should be usable to the GCS as well as logged for later in that use case. Any objection to the change if we don't send directions when they are loaded in as a proximity sensor? (Although this is getting annoyingly special cased at that point)